checker/jdk/nullness/src/java/util/ConcurrentMap.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package java.util;
```

Comment:
This is not the correct package for `ConcurrentMap`.  Why did you change it?
I have the same question about `ConcurrentHashMap`.